Source/Core/Core/CoreTiming.cpp

```diff
@@ -103,6 +97,13 @@ void CoreTimingManager::Init()
 
   m_event_fifo_id = 0;
   m_ev_lost = RegisterEvent("_lost_event", &EmptyTimedCallback);
+
+  m_registered_config_callback_id =
+      CPUThreadConfigCallback::AddConfigChangedCallback([this]() { RefreshConfig(); });
+  RefreshConfig();
+
+  m_last_oc_factor = m_config_oc_factor;
+  m_globals.last_OC_factor_inverted = m_config_oc_inv_factor;
```

Comment:
Why was this moved later?

Reply:
`RefreshConfig` now calls `UpdateSpeedLimit` with `GetTicks` which requires the timer members be initialized. I moved the timer member initialization earlier, but the diff makes it look like I moved a lot of stuff later.